hera_pspec/pspecdata.py

```diff
@@ -3213,10 +3302,34 @@ def raise_warning(warning, verbose=True):
 
 
 def _load_dsets(fnames, bls=None, pols=None, logf=None, verbose=True, 
```

Comment:
Wondering if these functions should maybe live in `utils.py` now, as `pspecdata.py` is becoming a beast.

Reply:
yeah, except for that they are still only used internally in `pspecdata.py`. If they become `load_dsets` and are 1) used by more than one module or 2) are useful as a standalone convenience function, then yeah probably should go into utils. Not sure if they are quite at that level yet though